src/hooks/index.ts

```diff
@@ -115,3 +115,49 @@ export const useUserFollowing = () : UserFollowing => {
 
     return { following, onFollow, onUnfollow };
 };
+
+type EventsFilter = {
+    later: ZetkinEvent[] | undefined;
+    today: ZetkinEvent[] | undefined;
+    tomorrow: ZetkinEvent[] | undefined;
+    week: ZetkinEvent[] | undefined;
+}
+
+export const useEventsFilter = (events : ZetkinEvent[] | undefined) : EventsFilter => {
```

Comment:
Is this hook being used for anything other than populating the `EventTabs` component? If not, I think this logic would make more sense inside that component.

Reply:
That's true, I don't think this needs to be reusable at this point.